ClientIdentifier.ts

```diff
@@ -0,0 +1,11 @@
+import { Identifier } from "authly"
+
+export namespace ClientIdentifier {
+
+	export function getClientId() {
+		let clientId = localStorage.getItem("clientId")
```

Comment:
Nothing is ever stored in localStorage.

Reply:
I thought that was the plan. How should it be stored?

Reply:
If you generate a new one make sure to store it.